tests/testthat/test-add-subvariable.R

```diff
@@ -91,3 +145,61 @@ with_test_authentication({
         )
     })
 })
+
+with_test_authentication({
+    # adding to derived array
+    # TODO: Use existing fixture? Add this to fixtures?
```

Comment:
This simple example works well for these tests, but we usually use `newDatasetFromFixture()`, worth changing to that (either by using existing fixture or adding this one there)?